FirebaseInstallations/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v1.1.0 -- M62.1
+
+- [changed] Throw an exception when there are missing required `FirebaseOptions` parameters (`APIKey`, `googleAppID`, and `projectID`). Please make sure your `GoogleServices-Info.plist` (or `FirebaseOptions` if you configure Firebase in code) are up to date. The file and settings can be downloaded from the [Firebase Console](https://console.firebase.google.com/).  (#4683)
```

Comment:
The code as it is now doesn't throw when `projectID ` is unavailable - so I think that should be removed.  

Also: `s/are up to date/is up to date/`

Reply:
done